public/styles/editor.css

```diff
@@ -442,14 +416,18 @@ body {
 
 /* WORLD */
 #world {
-	padding-top: 170px; /* For the inspector */
+	padding-top: 120px; /* For the inspector */
+	display: flex;
+	flex-direction: column;
 }
 
 	#worldHierarchy {
 		background: #111;
 		border: 3px solid #262626;
 		padding: 2px;
-
+		overflow-y: auto;
```

Comment:
Maybe if would be better to have the "Add Prefab" always visible, either on top or bottom of the hierarchy list.